spec/requests/edit_collection_spec.rb

```diff
@@ -27,6 +27,7 @@
       it 'allows GETs to /collections/{id}/edit' do
         get "/collections/#{collection.id}/edit"
         expect(response).to have_http_status(:ok)
+        expect(response.body).to include '<title>SDR | MyString</title>'
```

Comment:
not for this PR, but it would be useful to rename the title of the factory generated collection to at least "MyCollection"